src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/NHSD.BuyingCatalogue.Ordering.OrderingDatabase.sqlproj

```diff
@@ -101,4 +106,7 @@
       <Value>$(SqlCmdVar__1)</Value>
     </SqlCmdVariable>
   </ItemGroup>
-</Project>
+  <ItemGroup>
+    <None Include="Indexes\OdsCodeIndex.sql" />
+  </ItemGroup>
+</Project>
```

Comment:
Probably VS that has removed it but file should terminate with a new line.

Reply:
done